src/otx/algo/detection/heads/anchor_head.py

```diff
@@ -79,9 +77,9 @@ def __init__(
             raise ValueError(msg)
         self.reg_decoded_bbox = reg_decoded_bbox
 
-        self.bbox_coder = DeltaXYWHBBoxCoder(**bbox_coder)
-        self.loss_cls = CrossSigmoidFocalLoss(**loss_cls)
-        self.loss_bbox = GIoULoss(**loss_bbox)
+        self.bbox_coder = bbox_coder
+        self.loss_cls = loss_cls
+        self.loss_bbox = loss_bbox
         self.train_cfg = train_cfg
         self.test_cfg = test_cfg
         if self.train_cfg:
```

Comment:
Can we build ATSSAssigner and PseudoSampler outside in ssd_head or atss_head? For instance, 
```python
ATSSHead(
  assigner=ATSSAssigner(...),
  sampler=PseudoSampler(...),
  ...
)
```

Reply:
Thanks for your opinion, I changed to train_config hold assigner class. I think we can drop concept of train_cfg but I think  it is risky for our schedule.